### PR TITLE
Add translation files to all add-ons

### PR DESCRIPTION
**Pull request by ludeeus**
Add translation files were missing for all add-ons that have configuration or network options.
Some schema options were reordered to make the "flow" of the configuration better.

Example of the difference:
image
image

### CODE REVIEW

File: zwave/translations/en.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+configuration:
+  device:
+    name: Device
+    description: The device address of your Z-Wave controller.
```
Comment (agners) [inline zwave/translations/en.yaml:5]: Address sounds strange in that context. I know it comes from the docs, but still :sweat_smile: 

```suggestion
    description: The Z-Wave controller device.
```

File: zwave_js/translations/en.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+configuration:
+  device:
+    name: Device
+    description: The device address of your Z-Wave controller.
```
Comment (agners) [inline zwave_js/translations/en.yaml:5]:
```suggestion
    description: The Z-Wave controller device.
```

File: ada/translations/en.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+configuration:
```
[the remark this answers is not in the document]
Comment (agners) [reply ada/translations/en.yaml:2]: FWIW, it also is in other files inconsistent.

**Review by agners — CHANGES_REQUESTED** [review]

File: zwave/translations/en.yaml (hunk 1 above)
Comment (ludeeus) [reply zwave/translations/en.yaml:5]: Lifted from current documentation

File: zwave_js/translations/en.yaml (hunk 2 above)
Comment (ludeeus) [reply zwave_js/translations/en.yaml:5]: Lifted from current documentation

File: ada/translations/en.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (ludeeus) [reply ada/translations/en.yaml:2]: Does not matter at all, and if anything it should be `schema` not  `options`

File: mosquitto/translations/en.yaml [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      A file containing a root certificate. Place this file in the Home
+      Assistant `ssl` folder.
+  keyfile:
+    name: Private Key File
+    description: >-
+      A file containing the private key. Place this file in the Home Assistant
+      `ssl` folder.
+  customize:
+    name: Customize
+    description: Customize options.
```
Comment (agners) [inline mosquitto/translations/en.yaml:30]:
```suggestion
    description: See the Documentation tab for more information about these options.
```

File: nginx_proxy/translations/en.yaml [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    description: Private Private Key File to use in the `/ssl` directory.
+  cloudflare:
+    name: CloudFlare
+    description: >-
+      If enabled, configure Nginx with a list of IP addresses directly from
+      Cloudflare that will be used for `set_real_ip_from` directive Nginx
+      config.
+  customize:
+    name: Customize
+    description: Customize options.
```
Comment (agners) [inline nginx_proxy/translations/en.yaml:24]:
```suggestion
    description: See the Documentation tab for more information about these options.
```

**Review by agners — CHANGES_REQUESTED** [review]

File: mosquitto/translations/en.yaml (hunk 4 above)
Comment (ludeeus) [reply mosquitto/translations/en.yaml]: image

File: nginx_proxy/translations/en.yaml (hunk 5 above)
Comment (ludeeus) [reply nginx_proxy/translations/en.yaml]: image

File: zwave/translations/en.yaml (hunk 1 above)
Comment (agners) [reply zwave/translations/en.yaml:5]: I know, but its under a list of devices now. Probably worth updating the docs as well :)

File: ada/translations/en.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (agners) [reply ada/translations/en.yaml:2]: Right, `schema`.

I know that it doesn't matter, but its helpful later on, e.g. when adding new options or spot a missing translation.

**Review by agners — APPROVED** [review]

**Comment by mdegat01** [comment]
@ludeeus if you do a rebase here I believe CI will pass after #2449 . It was failing on the OOM exception for cryptography in Let's Encrypt and then skipping the rest.

**Comment by ludeeus** [comment]
Most likely

**Pull request merged**